templates/web/base/admin/roles/index.html

```diff
@@ -0,0 +1,36 @@
+[% INCLUDE 'admin/header.html' title=loc('Roles') -%]
+
+<table>
+    <tr>
+        <th>[% loc('Role') %]</th>
+        <th>[% loc('Permissions') %]</th>
+        <th>[% loc('Links') %]</th>
+    </tr>
+[% FOREACH role IN roles -%]
+  [% IF c.user.is_superuser AND last_name != role.body.name %]
+    <tr>
+        <td colspan="3"><strong>[% role.body.name %]</strong></td>
+    </tr>
+    [% SET last_name = role.body.name %]
+  [% END %]
+    <tr>
+        <td>[% role.name %]</td>
+        <td><ul class="no-margin no-bullets">
+          [% FOR perm IN role.permissions.sort %]
+            <li>[% labels.$perm %]
+          [% END %]
+        </ul></td>
+        <td>
+          <a href="[% c.uri_for(c.controller.action_for('edit'), [role.id]) %]">Edit</a>
+          &middot;
+          <a href="[% c.uri_for(c.controller.action_for('delete'), [role.id]) %]">Delete</a>
```

Comment:
This destructive action shouldn't be right alongside the edit link IMO. It'd be better as a big red button with a JS confirmation at the bottom of the role's edit page, like we do for e.g. templates:

![image](https://user-images.githubusercontent.com/4776/58245147-12541e80-7d4c-11e9-9071-1ae9fd842930.png)


We should indicate to the user how many users have this role before they delete it, maybe with a link to the pre-filled search page (`/admin/users?search=&role=1`) so they can verify who will be affected or re-assign roles etc.